fellow-submissions/devonte-duncan/styles.css

```diff
@@ -0,0 +1,55 @@
+html {
+  box-sizing: border-box;
+}
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+
+body{
+  width: 80%;
+  margin: 0 auto;
+}
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  display: flex;
+  align-self: center;
+}
+
+li{
+  margin: 10px;
+}
+
+header{
+  display: flex;
+  justify-content: space-around;
+}
+
+article{
+  display: flex;
+}
+
+main section:first-of-type{
```

Comment:
Not sure if you just wanted to experiment with the `:first-of-type` and `:only-of-type` pseudo selectors but they are unnecessary here. You put yourself at risk of "specificity creep" when you add additional class/pseudo class selectors, etc.